study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/RarexVersion3.java

```diff
@@ -0,0 +1,330 @@
+                + "            ";
+        updateVariableByText(consentParental, quitBefore, quitAfter);
+        updateVariableByText(consentAssent, quitBefore, quitAfter);
+
+        String eligibleBefore = "\n" 
+                + "              <p>Participants and families who may take part include:</p>\n" 
+                + "              <ul><li>Any person who has been diagnosed with a rare disease, or who is looking for a diagnosis.</li>\n" 
+                + "              <li>A parent or legal guardian of a child with a rare disease may register a child who is a minor (a "
+                + "“minor” is a child under the age of 18, in most states).</li>\n"
+                + "              <li>The legally authorized representative of an adult with a rare disease who cannot physically or "
+                + "mentally answer the surveys may enroll the affected participant.</li></ul>\n"
+                + "            ";
+        String eligibleAfter = "\n" 
+                + "              <p>Participants and families who may take part include:</p>\n" 
+                + "              <ul><li>Any person who has been diagnosed with a rare disease, or who is looking for a diagnosis.</li>\n" 
+                + "              <li>A parent or legal guardian of a child with a rare disease may register a child who is a minor"
+                + " (a “minor” is a child under the age of 18, in most states).</li>\n"
+                + "              <li>The legally authorized representative of an adult with a rare disease who cannot physically "
+                + "or mentally answer the surveys, may enroll the affected participant.</li></ul>\n"
+                + "            ";
+        updateVariableByText(consent, eligibleBefore, eligibleAfter);
+        updateVariableByText(consentParental, eligibleBefore, eligibleAfter);
+        updateVariableByText(consentAssent, eligibleBefore, eligibleAfter);
+        updateVariableByText(consentLar, eligibleBefore, eligibleAfter);
+        updateVariableByText(consentAssentLar, eligibleBefore, eligibleAfter);
+    }
+
+    private void updateExpressionTextByQuestionStableCode(long activityId, String questionStableCode, String expr) {
+        List<Long> ids = helper.findBlockExpressionIdsByQuestionStableCode(activityId, questionStableCode);
+        helper.updateExpressionText(expr, ids);
+    }
+
+    private void updateVariableByText(long activityId, String before, String after) {
+        List<Long> varIds = helper.findVariableIdsByText(activityId, before);
+        if (CollectionUtils.isEmpty(varIds)) {
+            throw new DDPException("Could not find any variable with text " + before);
+        }
+        varIds.forEach(varId -> {
+            helper.updateVarValueByTemplateVarId(varId, after);
+            LOG.info("Template variable {} text was updated from \"{}\" to \"{}\"", varId, before, after);
+        });
+    }
+
+    private interface SqlHelper extends SqlObject {
```

Comment:
looks like we have quite a few methods here in SqlHelper that are NOT used by this task, probably just cut  and pasted from RarexVersion2.java . Can we clean them up and remove unused code.

Reply:
Sure. Done